src/charts/js/CategoryAxis.js

```diff
@@ -86,10 +86,16 @@ Y.CategoryAxis = Y.Base.create("categoryAxis", Y.Axis, [Y.CategoryImpl], {
             data = this.get("data"),
             offset = edgeOffset;
         dataValues = dataValues || data;
+        var indexMap = new Map();
```

Comment:
It looks like we can source format this to add to the variable declarations above instead of declaring it in a new ```var``` attribute.